### PR TITLE
Add section on previous proof privacy consideration.

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to address issue #168 by highlighting a privacy consideration raised by PING and suggests a mechanism that could mitigate the concern.

/cc @kdenhartog

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/198.html" title="Last updated on Sep 25, 2023, 3:17 PM UTC (0fa2e00)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/198/6a3317a...0fa2e00.html" title="Last updated on Sep 25, 2023, 3:17 PM UTC (0fa2e00)">Diff</a>

### CODE REVIEW

**Review by kdenhartog — APPROVED** [review]
Lgtm good ideal to add a suggested mitigation strategy too.

**Review by dlongley — APPROVED** [review]

**Review by Wind4Greg — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -3577,6 +3577,32 @@ <h3>Selective Disclosure</h3>
...
 
+      <section>
+        <h3>Previous Proofs</h3>
+
+        <p>
+When utilizing the `previousProof` feature defined in
+<a href="#proof-chains"></a>, implementations are required to digitally sign
+over one or more previously specified proofs in order to include it in the
+secured payload. Doing so inevitably exposes information related to the
+entity that added the previous proof.
```
Comment (TallTed) [inline index.html]:
```suggestion
When using the `previousProof` feature defined in
<a href="#proof-chains"></a>, implementations are required to digitally sign
over one or more previous proofs, so as to include them in the
secured payload. This inevitably exposes information related to each
entity that added a previous proof.
```

File: index.html [hunk 2]
```diff
@@ -3577,6 +3577,32 @@ <h3>Selective Disclosure</h3>
...
+        </p>
+        <p>
+At a minimum, the <a>verification method</a> for the previous proof, such as a
+public key, is seen by the creator of the next proof in the proof chain. This
+can be a privacy concern if the creator of the previous proof was not intending
+to be included in a chain of proofs, but is an inevitable potential outcome when
+adding a non-repudiable digital signature on a document of any kind.
```
Comment (TallTed) [inline index.html]:
```suggestion
At minimum, the <a>verification method</a> for the previous proof, such as a
public key, is seen by the creator of the next proof in a proof chain. This
can be a privacy concern if the creator of the previous proof did not intend
to be included in a proof chain, but is an inevitable outcome when
adding a non-repudiable digital signature to a document of any kind.
```

File: index.html [hunk 3]
```diff
@@ -3577,6 +3577,32 @@ <h3>Selective Disclosure</h3>
...
+        </p>
+        <p>
+It is possible to utilize more advanced cryptographic mechanisms to hide the
+identity of the signer of a message, such as through the use of a
+<a href="https://en.wikipedia.org/wiki/Group_signature">group signature</a>
+and it is possible for a Data Integrity cryptographic suite to be created
+to mitigate the privacy concern raised in this section.
```
Comment (TallTed) [inline index.html]:
```suggestion
It is possible to use more advanced cryptographic mechanisms, such as a
<a href="https://en.wikipedia.org/wiki/Group_signature">group signature</a>,
to hide the identity of the signer of a message, 
and it is also possible for a Data Integrity cryptographic suite
to mitigate this privacy concern.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**